augmentedreality/Examples/CppArExample/CppArExample.pro

```diff
@@ -46,7 +59,7 @@ OTHER_FILES += \
 # The path to the ArcGIS runtime toolkit for Qt sources, corresponding to the files downloaded
 # from the GitHub repo: https://github.com/Esri/arcgis-runtime-toolkit-qt
 
-ARCGIS_TOOLKIT_PATH = # must be set to the root path of the toolkit folder
+ARCGIS_TOOLKIT_PATH = $$PWD/../../../ # must be set to the root path of the toolkit folder
```

Comment:
Was this intentional?

Reply:
Yes but only for the time being - will revert for the PR and keep it while in draft.